src/components/lists/lists.module.scss

```diff
@@ -0,0 +1,78 @@
+@import 'src/assets/stylesheets/base_styles/colors';
+@import 'src/assets/stylesheets/utilities/functions';
+@import 'src/assets/stylesheets/base_styles/typography';
+@import 'src/assets/stylesheets/base_styles/break_points';
+
+$article-list-color:                 $neutral-2;
+$article-list-heading-color:         $neutral-4;
+$article-list-heading-hover-color:   $neutral-1;
+$articles-list-heading-border-color: $action-2;
+$article-list-image-max-width:       ru(10);
+$articles-list-tag-color:            $neutral-3;
+
+.article-list {
+  list-style: none;
+  color: $article-list-color;
+}
+
+li {
+  margin-bottom: ru(1);
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+}
+
+.header {
+  @include like-h5;
+  margin: 0;
+  color: $article-list-heading-color;
+}
+
+p {
+  @include like-p;
```

Comment:
Same as above -- also you shouldn't be setting the font-family here -- this is something that should be set via the typography mixin.

Also, I think there's a strong enough argument against setting styles on HTML elements that we should follow it and use classes only.